src/common/components/paymentMethods/creditCardForm/creditCardForm.component.spec.js

```diff
@@ -127,6 +129,7 @@ describe('credit card form', () => {
     });
     it('should send a request to save the credit card payment and billing address info', () => {
       self.controller.creditCardPayment = {
+        'address': formatAddressForCortex(self.controller.billingAddress),
```

Comment:
Was this test not working without this? https://github.com/CruGlobal/give-web/blob/master/src/common/components/paymentMethods/creditCardForm/creditCardForm.component.js#L112 should fill it in depending on if we are using mailing address or billing address.